Makefile

```diff
@@ -0,0 +1,33 @@
+DOCKER_IMAGE = oam/server:latest
+
+all: server
+
+server:
+	@docker build -f ./server/Dockerfile -t $(DOCKER_IMAGE) ./server
+
+start: server
+	@docker run \
+		--detach \
+		--name oam-server \
+		--publish 8000:8000 \
+		--volume $(PWD)/server:/app \
+		$(DOCKER_IMAGE) start
+
+test: start
+	@sleep 1
+
+	@docker run \
+		--rm \
+		--name oam-server-test \
+		--link oam-server:oam-server \
+		--volume $(PWD)/server:/app \
```

Comment:
If we are doing a `COPY` of the whole directory as part of the build, do we need to link the folders?